packages/client/src/utils/validate.ts

```diff
@@ -550,10 +549,7 @@ export const validIDNumber =
       }
 
       return {
-        message: messages.validNationalId,
-        props: {
-          validLength: num
-        }
+        message: messages.validNationalId
       }
     }
     return undefined
```

Comment:
`duplicateIDNumber` will have to be changed.
`if (value && valueToCheck && +value === +valueToCheck) ` Change this line.

Reply:
This code can be found in latest `develop` branch. @emon-swe-sust please test the `duplicateIDNumber` in `correct record`.